hack/make-rules/test-integration.sh

```diff
@@ -85,6 +85,7 @@ checkEtcdOnPath() {
   which etcd && return
   kube::log::status "Cannot find etcd, cannot run integration tests."
   kube::log::status "Please see docs/devel/testing.md for instructions."
```

Comment:
I'd rather just update this link to https://github.com/kubernetes/community/blob/master/contributors/devel/testing.md#install-etcd-dependency as there are extra steps, including adding it to the path.